auto_submit/lib/model/big_query_pull_request_record.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class PullRequestRecord {
+  PullRequestRecord({
+    this.prCreatedTimestamp,
+    this.prLandedTimestamp,
+    this.organization,
+    this.repository,
+    this.author,
+    this.prId,
+    this.prCommit,
+    this.prRequestType,
+  });
+
+  int? prCreatedTimestamp;
```

Comment:
Can we make this `DateTime`? Making timestamps ints is a pain to process in SQL

Reply:
They aren't timestamps in bigquery. I was told we use ints for processing reasons.

Reply:
I'll process them in the code.

Reply:
Done. These are only ints in the database.